frontend/javascripts/admin/organization/organization_cards.tsx

```diff
@@ -9,12 +9,67 @@ import { Alert, Button, Card, Col, Progress, Row } from "antd";
 import { formatDateInLocalTimeZone } from "components/formatted_date";
 import { useFetch } from "libs/react_helpers";
 import moment from "moment";
+import Constants from "oxalis/constants";
 import React from "react";
 import { APIOrganization } from "types/api_flow_types";
 import { PricingPlanEnum } from "./organization_edit_view";
-import { powerPlanFeatures, teamPlanFeatures } from "./pricing_plan_utils";
+import {
+  hasPricingPlanExceededStorage,
+  hasPricingPlanExceededUsers,
+  hasPricingPlanExpired,
+  powerPlanFeatures,
+  storageWarningThresholdMB,
+  teamPlanFeatures,
+} from "./pricing_plan_utils";
 import UpgradePricingPlanModal from "./upgrade_plan_modal";
 
+export function TeamAndPowerPlanUpgradeCards({
+  teamUpgradeCallback,
+  powerUpgradeCallback,
+}: {
+  teamUpgradeCallback: () => void;
+  powerUpgradeCallback: () => void;
+}) {
+  return (
+    <Row gutter={24}>
+      <Col span={12}>
+        <Card
+          title={`${PricingPlanEnum.Team} Plan`}
+          bodyStyle={{ minHeight: 220, opacity: 0.8 }}
+          actions={[
+            <Button type="primary" onClick={teamUpgradeCallback}>
+              <PlusCircleOutlined /> Request Upgrade
+            </Button>,
+          ]}
+        >
+          <ul>
+            {teamPlanFeatures.map((feature) => (
+              <li key={feature.slice(0, 10)}>{feature}</li>
```

Comment:
why the slicing? do we have that many features? and is it ok to crop them without having the chance to click sth like "show more"?

Reply:
if you want to keep it, i'd suggest to extract the 10 to a constant. I think, it's used at least at 3 locations.

Reply:
I don't remember the reason. Maybe because we had some mock data at some point. Will remove.

Reply:
Ah... I think I wanted to have "short", unique keys. The slicing is called on a string, not an array.